llvm/docs/DeveloperPolicy.rst

```diff
@@ -104,6 +104,12 @@ When submitting patches, please do not add confidentiality or non-disclosure
 notices to the patches themselves.  These notices conflict with the LLVM
 licensing terms and may result in your contribution being excluded.
 
+When being asked by someone to add code (e.g. create a PR), or write review
+comments on their behalf, do suggest that they do it directly themselves. Or
+ask the reasons why they're not doing it themselves. There should be few
+legitimate reasons for them not doing so themselves.
+
```

Comment:
Maybe encourage reaching out to the coc team if you suspect that something fishy is going on after asking.

Reply:
I'm struggling to find a way to phrase it, but I think we could do with a statement about the level of due diligence that the person doing the action on behalf of has to do. As there are plausible excuses that could be given by a banned person.

I think adding the statement about the CoC team would help here. So that this is a process that has clear steps. Ask, verify as you can, forward anything untoward to the CoC team.

I wouldn't want people (including me) to read this and draw the conclusion that it's not worth the risk to merge anything on behalf of. Especially as it could lead to a situation where these merges are more and more done by people who are not going to be as attentive.

Not intending this to be some kind of immunity statement, I am ok with merging anyone else's code having some level of risk to it.

Reply:
Actually I think I am conflating a few things here. This text does not refer to merging an existing PR, but creating your own PR with someone else's code.

So you can ignore most of that, my apologies.

I still think a clearer statement of steps would be good.